fedimint-client/src/lib.rs

```diff
@@ -742,6 +746,7 @@ pub struct Client {
     root_secret: DerivableSecret,
     operation_log: OperationLog,
     secp_ctx: Secp256k1<secp256k1_zkp::All>,
+    meta_manager: Arc<MetaManager>,
```

Comment:
I'm not a fan of managers. :D . Would name with something more concrete.

Reply:
@coderabbitai please suggest a better name